src/app/exams/exams-view.component.ts

```diff
@@ -51,7 +53,7 @@ export class ExamsViewComponent implements OnInit, OnDestroy {
   }
 
   goBack() {
-    this.router.navigate([ '../../' ], { relativeTo: this.route });
+    this.location.back();
```

Comment:
this code can be reverted so exam will take back to step

Reply:
Ok i will revert the mention changes but if i click take test from course view then it will send back to step view